### ISSUE TITLE
Trouble with installation

### ISSUE THREAD

**Issue by jamesallison** [MengTo/Spring#145]
I've dropped the Spring folder into my project, with copying and groups enabled. when I import spring it says "No such module 'Spring'".

**Comment by NiranjanRavichandran** [comment]
I don't think it is required to import the framework unless you have used cocoa pods - "use_framework!". If you have just dragged and dropped into your project it should be accessible throughout your projects since all files are swift files. Hope this helps.

**Comment by jamesallison** [comment]
I've tried not importing Spring, however still self.labelName.layer.animation = "squeezeDown" errors with 

> "Value of type CALayer has no member 'animation'"

**Comment by netgfx** [comment]
Same thing here, can't link Spring or .animation
Installed Spring through Carthage

**Comment by apprisingapps** [comment]
same issue.  I have tried both through cocoa pods, and just dragging and dropping without the import.  Was any solution found?
